### PR TITLE
enable GCWhere on PAL

### PR DESCRIPTION
**Pull request by Maoni0**
This is a simple but useful command, enabling for PAL.

### CODE REVIEW

**Comment by Maoni0** [comment]
@mikem8361 @swgillespie PTAL

File: src/ToolBox/SOS/Strike/strike.cpp [hunk 1]
```diff
@@ -9730,6 +9728,8 @@ DECLARE_API(GCWhere)
     return Status;
 }
 
+#ifndef FEATURE_PAL
+
 DECLARE_API(FindRoots)
 {
 #ifndef FEATURE_PAL
```
Comment (swgillespie) [inline src/ToolBox/SOS/Strike/strike.cpp]: not your code but won't this always be defined here?

**Comment by swgillespie** [comment]
LGTM

**Comment by mikem8361** [comment]
LGTM

File: src/ToolBox/SOS/Strike/strike.cpp (hunk 1 above)
Comment (Maoni0) [inline src/ToolBox/SOS/Strike/strike.cpp]: Yep, there are a few nested #ifndef FEATURE_PAL here. this code could use some cleanup...

Comment (mikem8361) [inline src/ToolBox/SOS/Strike/strike.cpp]: Don't worry about it now.

Comment (mikem8361) [inline src/ToolBox/SOS/Strike/strike.cpp]: I noticed you didn't include the CMakeFiles.txt change. Would you rather I get that in?

Comment (Maoni0) [inline src/ToolBox/SOS/Strike/strike.cpp]: oh, I thought you wanted to check it in. I can include it.

Comment (Maoni0) [inline src/ToolBox/SOS/Strike/strike.cpp]: Added the fix Mike gave me in cmakelists.txt

Comment (mikem8361) [inline src/ToolBox/SOS/Strike/strike.cpp]: Sorry, I wasn’t clear before but since your PR is ready now I do the CMakeFile.txt fix.

Comment (mikem8361) [inline src/ToolBox/SOS/Strike/strike.cpp]: Oops, now my posts are way behind. Your CMakeLists.txt are good. You are good to go. Thanks.

**Pull request merged**